### ISSUE TITLE
[ACCESSIBILITY] Visual focus is not given properly to items as screen reader user swipes through 

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#1495]
**Describe the accessibility issue**
Visual focus is not given properly to items as screen reader user swipes through. 
In the event a low sighted user is using screen reader software it is important that the visual focus being given to items lines up with the correct items so that user may select properly. See screenshots for understanding of how this experience can affect the users of assistive technology. 

[WCAG 1.4.13 Content on Hover or Focus (AA)](https://www.w3.org/TR/WCAG21/#content-on-hover-or-focus)
[WCAG 2.4.3 Focus Order (A)](https://www.w3.org/TR/WCAG21/#focus-order)
[WCAG 2.4.7 Focus Visible (AA)](https://www.w3.org/TR/WCAG21/#focus-visible)

**Steps to reproduce the behavior:**
_SEE EXAMPLES BELOW --> Entire app has this issue, hard to describe steps where to find. Start with FAQ pages for COVID or building access._ 

**Expected behavior**

**Screenshots**
GOOD EXAMPLE:
![Screenshot_20220308-093945](https://user-images.githubusercontent.com/82892873/160191609-6c6a69f1-b9f9-4990-8091-1563a14ecd9c.png)

**VIOLATIONS: (NOTE --> THERE ARE MANY MORE THAN JUST LISTED HERE, it is suggested to review app with screen reading software prior to release to ensure the visual focus indicators are where expected.**

![Screenshot_20220308-142625](https://user-images.githubusercontent.com/82892873/160191805-4480d5d8-0711-4e43-8c7d-a5686fe041a9.png)

![Screenshot_20220308-142638](https://user-images.githubusercontent.com/82892873/160191816-e4034457-dfce-4df2-b8b4-778bf8c98217.png)

![Screenshot_20220308-142710](https://user-images.githubusercontent.com/82892873/160191828-aa1ee4fc-b550-46b8-a9a6-e7d1f3a04f72.png)

**Smartphone (please complete the following information):**
iPhone XR- iOS 14.6, Pixel 4a- Android 12

**Additional context**

**Comment by TodorBachvarov** [comment]
Unfortunately we can't fix this. All given examples and probably all the places it's observed is caused by the HTMLWidget that we use and it's known for its poor Accessibility support.

**Comment by acovher2** [comment]
This needs to be discussed, I'm not sure this is acceptable, we may be subject to compliance lawsuits. Stay tuned please.

**Comment by acovher2** [comment]
Per Mark, 

[InabyteMark](https://app.slack.com/team/UFT217AAH)
  [3:10 PM](https://rokwire-workspace.slack.com/archives/C035Y481S5T/p1657570211209319)
On 1455, I should have left you a comment in GIT as this was moved back to me when we had started working the 4.0 release and were not sure what was going to happen with all the changes being made. We have started looking at all GIT issues and are now actively working on what we can now and for the rest of this week so we should be able to do something here. The same goes for 1495 but for July 15th it is unlikely we will find a replacement plugin for the Flutter HTML control. I will raise our looking for a solution for this one to the Aug 15 release.[InabyteMark](https://app.slack.com/team/UFT217AAH)

**Comment by acovher2** [comment]
Hello, 
An update, we should not be using any third party widgets that are knowingly inaccessible. This leaves the institution subject to lawsuits. We need a new widget for this asap that is accessible, whether that means making a custom one or finding an accessible pre-existing one. 
Please escalate this issue to top of accessibility list. Thank you.

**Comment by acovher2** [comment]
@pmarkhennessy I was awaiting solution for this in August and upon retesting the December release this is still not included. Please provide a status as to where this bug is. thank you.

**Comment by acovher2** [comment]
This has been ignored too long. It is a showstopper and needs resolution. App is not in compliance and we are in precarious situation accessibility-wise.

**Comment by acovher2** [comment]
Test Pass. This is still not perfect, but MUCH improved experience for the user. Closing.
